src/icons/index.tsx

```
@@ -1,19 +1,31 @@
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox={"0 0 24 24"}
+      width={size}
+      height={size}
+      className={className}
+      fill={solid ? color : "none"}
+      stroke={color}
+      strokeWidth="1"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+      {...props}
+      dangerouslySetInnerHTML={{ __html: icon.symbol }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)